src/styles/Contributors.module.scss

```
@@ -0,0 +1,130 @@
+@import '_global';
```

Stylelint (17.13.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)